### PR TITLE
Adding steps for custom vagrantfiles

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Adding steps for custom vagrantfiles and more information about installation steps of ADB box

Signed-off-by: Lalatendu Mohanty lmohanty@redhat.com

### CODE REVIEW

File: docs/installing.rst [hunk 1]
```diff
@@ -136,3 +140,25 @@ There are two ways to download the ADB.  You can have ``vagrant`` do it for you
...
     $ vagrant box add atomicappbox <local path to the downloded image>
+    $ vagrant init atomicappbox
+    $ vagrant up
+
+
+-------------------
+4. Using custom vagrantfiles for specific use cases
+-------------------
+
+There are custom vagrantfiles at `components directory <../components>` which can be used for creating specific environments.
```
Comment (navidshaikh) [inline docs/installing.rst]: Local reference to `components` directory here would be good.

Comment (LalatenduMohanty) [inline docs/installing.rst]: @navidshaikh yup, the components directory is hyper linked here.

Comment (LalatenduMohanty) [inline docs/installing.rst]: Ahh, the hyperlink is not working :(. Will resubmit

**Comment by navidshaikh** [comment]
LGTM

**Pull request merged**